Makefile

```diff
@@ -131,3 +132,32 @@ env_restart:
 env_clean: env_down
 	@echo "=> Cleanup environment"
 	@docker volume rm docker_volume_chain
+
+# --- Windows Server Core environment ---
+env_image-wsc:
```

Comment:
Can we detect Windows and handle this in the `env_image` target?

Reply:
Fixed!